test/v1beta1/occurrence_test.go

```diff
@@ -128,6 +77,119 @@ func TestOccurrence(t *testing.T) {
 	})
 
 	t.Run("listing occurrences", func(t *testing.T) {
-		t.Skipf("TODO")
+		// setup project specifically for listing occurrences
+		listProjectName := util.RandomProjectName()
+
+		_, err := util.CreateProject(s, listProjectName)
+		Expect(err).ToNot(HaveOccurred())
+
+		// creating three different occurrences to test against
+		occurrences := []*grafeas_go_proto.Occurrence{
+			createFakeBuildOccurrence(listProjectName),
+			createFakeVulnerabilityOccurrence(listProjectName),
+			createFakeAttestationOccurrence(listProjectName),
+		}
+
+		// ensure the first two occurrences have something in common to filter against
+		occurrences[0].Resource.Uri = occurrences[1].Resource.Uri
+
+		// create
+		_, err = s.Gc.BatchCreateOccurrences(s.Ctx, &grafeas_go_proto.BatchCreateOccurrencesRequest{
+			Parent:      listProjectName,
+			Occurrences: occurrences,
+		})
+		Expect(err).ToNot(HaveOccurred())
+
+		t.Run("should be successful", func(t *testing.T) {
```

Comment:
I think the listing occurrences and filtering test could be done similar to projects test https://github.com/liatrio/grafeas-elasticsearch/blob/main/test/v1beta1/project_test.go#L26.

You can still have crafted setup more matches or not, and iterate of filters and expected output. Makes it easier if we add more filtering logic in the future.